hip/test/matrix/coo_kernels.hip.cpp

```diff
@@ -271,4 +271,26 @@ TEST_F(Coo, ExtractDiagonalIsEquivalentToRef)
 }
 
 
+TEST_F(Coo, InplaceAbsoluteMatrixIsEquivalentToRef)
+{
+    set_up_apply_data();
+
+    mtx->compute_absolute_inplace();
+    dmtx->compute_absolute_inplace();
+
+    GKO_ASSERT_MTX_NEAR(mtx.get(), dmtx.get(), 1e-14);
```

Comment:
(Comment for all tests in general) I believe you don't need the .get() here, it gets removed by `gko::test::assertions::detail::plain_ptr;` internally, as far as I know

Reply:
I only refine the modification part in this PR.